.github/workflows/tests.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on: [push, pull_request]
+  # push:
+  #   branches: [ master ]
+  # pull_request:
+  #   branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install .[sphinx,code_style,testing]
+    - name: Pre-commit checks
+      run: |
+        pre-commit run --all-files || ( git status --short ; git diff ; exit 1 )
+    # cd tests/test_commonmark && ./spec.sh
+    - name: Test with pytest
+      run: |
+        pip install pytest
+        pytest --cov=myst_parser --cov-report=
+    - name: Upload to coveralls
+      run: |
+        pip install coveralls
+        coveralls
+      env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_KEY }}
+
+  publish:
```

Comment:
how is a publish vs. a build triggered? this seems like a useful way to automate releases, just not sure how it works :-)